documentation/library_guide/kernel_templates/esimd/radix_sort.rst

```diff
@@ -1,37 +1,37 @@
 Radix Sort
 ##########
 
----------------------------------------------------
-radix_sort and radix_sort_by_key Function Templates
----------------------------------------------------
+-----------------------------
+radix_sort Function Templates
+-----------------------------
 
-The ``radix_sort`` and ``radix_sort_by_key`` functions sort data using the radix sort algorithm.
+The ``radix_sort`` function sorts data using the radix sort algorithm.
 The sorting is stable, ensuring the preservation of the relative order of elements with equal keys.
 The functions implement a Onesweep* [#fnote1]_ algorithm variant. Both in-place and out-of-place
 overloads are provided. For out-of-place overloads, the input data order is preserved.
```

Comment:
I do not understand why "data order" is mentioned; it's a bit confusing. I believe the intended message is that the input data sequence is not affected by sorting.
For example, the documentation of CUB states it much clearer:
> The contents of the input data are not altered by the sorting operation.

Reply:
I clarified it. I also simplified the summaries of both algorithms. I hope it became more clear overall.